### DISCUSSION TITLE
[Question]TextArea 입력글자수에 콤마 안생기는 이슈

### DISCUSSION THREAD

**Discussion by yangSangHoon** [wanteddev/montage-web#20] [Q&A]
### 요약

textArea글자수가 1000자가 넘어도 콤마가 안생겨서 문의 드립니다.

### 상세

![Screenshot 2024-05-27 at 2 55 38 PM](https://github.com/wanteddev/wds/assets/15603335/5db3607b-9474-4751-b275-be05ab189ee9)

**Comment by Sh031224** [comment]
다음 버전에서 수정될 예정입니다. #21
